core/test/network-records-to-devtools-log.js

```diff
@@ -5,13 +5,19 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
+import assert from 'assert/strict';
+
 import {NetworkRecorder} from '../../core/lib/network-recorder.js';
 
 /** @typedef {import('../../core/lib/network-request.js').NetworkRequest} NetworkRequest */
 
 const idBase = '127122';
 const exampleUrl = 'https://testingurl.com/';
 const redirectSuffix = ':redirect';
+// Default request startTime if none provided. Do not use 0 due to guard against
+// bad network records starting at 0. See https://github.com/GoogleChrome/lighthouse/pull/6780
+const defaultStart = 1000;
+const defaultTimingOffset = 1000;
```

Comment:
these two values just allow us to be lazy when making mock requests and the results will look reasonable, right?
If the true numbers arent provided, then the req starts at 1000 and then two of the phases take at least 1s.. ya?

Reply:
yep, exactly. I left a comment saying basically that down on `getNormalizedRequestTiming` but could put something up here as well if it would help